components/sections/FAQSection.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+import { HelpCircle } from "lucide-react";
+import content from "@/app/content";
+
+export function FAQSection() {
+  return (
+    <section id="faq" className="w-full py-12 md:py-24 lg:py-32 border-t bg-muted/50">
+      <div className="container px-4 md:px-6">
+        <div className="flex flex-col items-center justify-center space-y-4 text-center mb-12">
+          <h2 className="text-3xl font-bold tracking-tighter sm:text-4xl">
+            {content.faq.title}
+          </h2>
+        </div>
+        <div className="mx-auto max-w-3xl">
+          <Accordion type="single" collapsible className="w-full">
+            {content.faq.questions.map((faq, index) => (
+              <AccordionItem key={index} value={`item-${index + 1}`}>
```

Comment:
also apply the same strategy for name that you used for the value but in key

```suggestion
              <AccordionItem key={`faq-item-${index}`} value={`item-${index + 1}`}>
```